specification/storagesync/resource-manager/Microsoft.StorageSync/StorageSync/CloudEndpoint.tsp

```diff
@@ -74,6 +76,37 @@ interface CloudEndpoints {
     Error = StorageSyncError
   >;
 
+  /**
+   * Patch a given CloudEndpoint.
+   */
+  @added(Versions.v2025_12_01)
+  @patch(#{ implicitOptionality: false })
+  @Azure.Core.useFinalStateVia("location")
+  update is Legacy.CustomPatchAsync<
+    CloudEndpoint,
+    PatchModel = CloudEndpointUpdateParameters,
+    Response = (ArmResponse<CloudEndpoint> & {
+      @header("x-ms-correlation-request-id")
+      @doc("correlation request id.")
+      correlationRequestId?: string;
+
+      @header("x-ms-request-id")
+      @doc("request id.")
+      requestId?: string;
+    }) | (ArmAcceptedLroResponse<LroHeaders = ArmCombinedLroHeaders<FinalResult = CloudEndpoint> &
+      Azure.Core.Foundations.RetryAfterHeader> & {
+      @header("x-ms-correlation-request-id")
+      @doc("correlation request id.")
+      correlationRequestId?: string;
+
+      @header("x-ms-request-id")
+      @doc("request id.")
+      requestId?: string;
+    }),
+    Error = StorageSyncError,
+    OptionalRequestBody = true
```

Comment:
For `PATCH` operations, the request body is generally required. Please make sure whether setting `OptionalRequestBody = true` is appropriate.